packages/mds-web-sockets/clients.ts

```diff
@@ -2,12 +2,33 @@ import WebSocket from 'ws'
 import { WebSocketAuthorizer } from '@mds-core/mds-api-authorizer'
 import { AuthorizationError } from '@mds-core/mds-utils'
 import log from '@mds-core/mds-logger'
+import jwt, { GetPublicKeyOrSecret } from 'jsonwebtoken'
+import jwks from 'jwks-rsa'
+import { promisify } from 'util'
 
 export class Clients {
   authenticatedClients: WebSocket[]
 
   subList: { [key: string]: WebSocket[] }
 
+  private static getKey: GetPublicKeyOrSecret = async header => {
+    const client = jwks({
+      jwksUri: 'ourIssuer'
```

Comment:
Curious if this is the real `jwksUri` we're using or if it will be read in from config in the future

Reply:
Will be read in from config in the future. I need to sync up with alex to see how trivial it is to get all of those variables passed down from the istio layer where they currently live. Same for a bunch of other variables which are just placeholders for now.